dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +283,17 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name='build-docs')
+@click.option('--docs-only', is_flag=True)
+@click.option('--spellcheck-only', is_flag=True)
+@click.option('--package-filter', type=click.Choice(AVAILABLE_PACKAGES))
```

Comment:
There is a small problem with the choices though as they allow also some other values. Package-filter can take values like `apache-airflow-provider-*`. And Click has no support for "list" autocomplete with some additional options available. Though I think this is arguable if it is useful or not. I don't think I ever used it to be honest. 

I am inclined to skip the 'wildcard' capability and leave the list of packages only here.

WDYT @mik-laj ? FYI: this is the wrapper around the "build_docs.py" - with much simpler interface, executing the build always inside the breeze container and autocomplete in Breeze so having fixed set of packages might make sense actually. Maybe we can use `--package` instead of `--package-filter` to indicate that those are only lists of packages and not "filters" as it is in case of `build-docs.py. I find that most of the 'casual" users would likely only use packages and autocomplete there would be **really** useful. Where we can keep `./build_docs.py` for Power users with filtering etc. 

WDYT @mik-laj